jbpm-case-mgmt/jbpm-case-mgmt-api/src/main/java/org/jbpm/casemgmt/api/model/instance/CaseInstance.java

```diff
@@ -66,6 +66,8 @@
      */
     CaseFileInstance getCaseFile();
     
+    void setCaseFile (CaseFileInstance  caseFileInstance);
```

Comment:
Is this really needed to be exposed in the model? Other setters are not part of the interface.
Missing doc comment above.

Reply:
Either we add this or we have to cast when setting the case file in case "withData" is true, I will incorporate the java doc.